ios/TransactionListView.swift

```diff
@@ -57,7 +57,23 @@ class TransactionListView: UIView, UITableViewDelegate, UITableViewDataSource {
       header.accountNameViewWidthConstraint.constant = accountAddressWidth
     }
   }
-  
+  @objc var accountImage: NSString? = nil {
+   didSet {
+    if (accountImage != nil) {
+      let url = URL.init(fileURLWithPath: accountImage?.expandingTildeInPath ?? "")
```

Comment:
Assuming the account image is not `nil`
```suggestion
      let url = URL.init(fileURLWithPath: accountImage!.expandingTildeInPath)
```